business/istio_validations.go

```diff
@@ -463,6 +466,15 @@ func (in *IstioValidationsService) fetchDetails(rValue *kubernetes.IstioDetails,
 			}
 			go fetchIstioObjects(&istioDetails.Sidecars, namespace, getSidecars, &wg2, errChan2)
 		}
+		if nsCached && kialiCache.CheckIstioResource(kubernetes.RequestAuthentications) {
```

Comment:
This is good.
No blocker, probably good to have a follow up PR to add all validations objects to the cache.
For Istio, only it's cached VirtualServices,DestinationRules,Gateways and Sidecars, perhaps we can add more.
(It's also a tradeoff).

I can prepare a next PR for that, too.